test/built-ins/Object/fromEntries/iterator-closed-for-throwing-entry-value-accessor.js

```diff
@@ -0,0 +1,64 @@
+// Copyright (C) 2018 Kevin Gibbons. All rights reserved.
```

Comment:
Presumably this (and other new tests) should be (C) someone else?

Reply:
Well, I didn't really write a majority of the test, it's your work with only a slight modification